contrib/scripts/start-crc-bundle.sh

```diff
@@ -4,16 +4,14 @@ set -exu
 
 YQ=${YQ:-yq}
 BUNDLE_PATH=$1
-KERNEL=$(cat ${BUNDLE_PATH}/crc-bundle-info.json | ${YQ} .nodes[0].kernel)
-INITRD=$(cat ${BUNDLE_PATH}/crc-bundle-info.json | ${YQ} .nodes[0].initramfs)
-CMDLINE=$(cat ${BUNDLE_PATH}/crc-bundle-info.json | ${YQ} .nodes[0].kernelCmdLine)
+EFISTORE_PATH="$HOME/.crc/machines/crc/efistore.nvram"
 DISKIMG=$(cat ${BUNDLE_PATH}/crc-bundle-info.json | ${YQ} .storage.diskImages[0].name)
 cp -c ${BUNDLE_PATH}/${DISKIMG} overlay.img
+cp -c ${EFISTORE_PATH} efistore.nvram
+
 
 ./out/vfkit --cpus 2 --memory 2048 \
-    --kernel "${BUNDLE_PATH}/${KERNEL}" \
-    --initrd "${BUNDLE_PATH}/${INITRD}" \
-    --kernel-cmdline "${CMDLINE}" \
+    --bootloader efi,variable-store=efistore.nvram,create \
```

Comment:
`create` will ensure that `efistore.nvram` is created, we don't need to reuse `"$HOME/.crc/machines/crc/efistore.nvram"` which may not exist
Using a new efi variable store will ensure we are as close as possible to a 'blank state' for our VM.

Reply:
ah, i misunderstood your prev comment. Updated